ui/js/selectors/claims.js

```diff
@@ -121,15 +121,26 @@ export const selectIsFetchingClaimListMine = createSelector(
   state => !!state.isFetchingClaimListMine
 );
 
-export const selectMyClaimsRaw = createSelector(
+export const cantFigureOutVarName = createSelector(
```

Comment:
@kauffj I couldn't figure out the variable name(it is basically `selectMyClaimsRaw` but that is already used. Two things could be done, either rename `selectMyClaimsRaw` at every place or find a good/better a name for `cantFigureOutVarName`)

Reply:
My proposal:

`cantFigureOutVarName` -> `selectMyClaims`
`selectMyClaimsRaw` -> `selectMyActiveClaims` (or another word that says the difference between this and `selectMyClaims` is the ones pending abandon are filtered)
`selectMyClaims` -> `selectMyPublishClaims`
`selectAllMyClaimsByTxidNout` -> `selectMyClaimsByOutpoint`

Reply:
Counter Proposal:
`cantFigureOutVarName` -> `selectMyClaimRaw`
`selectMyClaimsRaw` -> `selectMyActiveClaims`
`selectMyClaims` -> No Change
`selectAllMyClaimsByTxidNout` -> `selectAllMyClaimsByOutpoint`